### PR TITLE
Add Expeditor::Service#fallback_enabled

### PR DESCRIPTION
**Pull request by makimoto**
Implement "3. Fallbackable flag" of #8. ("Fallbackable is renamed to "fallback_enabled")
It provides "enabling and disabling" fallback blocks in the command.

@cookpad/dev-infra 👓 ?

### CODE REVIEW

**Comment by k0kubun** [comment]
Tests are broken.

File: spec/expeditor/service_spec.rb [hunk 1]
```diff
@@ -87,4 +87,37 @@
...
+
+    context 'fallback_enabled is false' do
+      before do
+        service.fallback_enabled = false
+      end
+
+      it 'does not call fallback and raises error' do
+        expect { Expeditor::Command.new(service: service) {
+          raise 'error!'
+        }.with_fallback {
```
Comment (k0kubun) [inline spec/expeditor/service_spec.rb]: With this indentation, this `}` seems to match `expect {`'s `{`.

I prefer:

```rb
expect {
  Expeditor::Command.new(service: service) {
    raise 'error!'
  }.with_fallback {
    0
  }.start.get 
}.to raise_error(RuntimeError, 'error!')
```

Comment (makimoto) [inline spec/expeditor/service_spec.rb]: fixed in 5ddb69e

**Comment by makimoto** [comment]
> Tests are broken.

oops. fixed in 684c4f5.

**Comment by k0kubun** [comment]
Conflicted with your other changes. Could you resolve it and confirm tests pass?

**Comment by makimoto** [comment]
done.

File: lib/expeditor/command.rb [hunk 2]
```diff
@@ -43,7 +43,7 @@ def started?
     def get
       raise NotStartedError if not started?
       @normal_future.get_or_else do
-        if @fallback_var
+        if @fallback_var && @service.fallback_enabled?
```
Comment (k0kubun) [inline lib/expeditor/command.rb]: In this file, `@fallback_var` is referenced in other places. Are you sure `@service.fallback_enabled?` should affect only this `@fallback_var`?

Comment (makimoto) [inline lib/expeditor/command.rb]: I found another point it affects fallback. Fixed it in 11e55be636907e537e9ab4769a55d115a9e42b89.

**Comment by adorechic** [comment]
LGTM

**Comment by k0kubun** [comment]
LGTM

**Pull request merged**